plugin.js

```diff
@@ -37,14 +41,14 @@ Plugin.registerSourceHandler("less", {archMatching: 'web'}, function (compileSte
   if ( config.useIndex ) {
     var indexFilePath = config.indexFilePath || DEFAULT_INDEX_FILE_PATH;
     // If this isn't the index file, add it to the index if need be
-    if ( compileStep.inputPath != indexFilePath ) {
+    if ( convertToPosixPath(compileStep.inputPath) != indexFilePath ) {
       if ( fs.existsSync(indexFilePath) ) {
         var lessIndex = fs.readFileSync(indexFilePath, 'utf8');
-        if ( lessIndex.indexOf(compileStep.inputPath) == -1 ) {
-          fs.appendFileSync(indexFilePath, '\n@import "' + compileStep.inputPath + '";', 'utf8');
+        if ( lessIndex.indexOf(convertToPosixPath(compileStep.inputPath)) == -1 ) {
+          fs.appendFileSync(indexFilePath, '\n@import "' + convertToPosixPath(compileStep.inputPath) + '";', 'utf8');
         }
       } else {
-        var newFile = generatedMessage + '@import "' + compileStep.inputPath + '";\n';
+        var newFile = generatedMessage + '@import "' + convertToPosixPath(compileStep.inputPath) + '";\n';
         fs.writeFileSync(indexFilePath, newFile, 'utf8');
```

Comment:
Thanks for the pull request! What's the platform-independent way of writing file paths? Does Windows get mad reading the `main.less` file if it's filled with a bunch of posix-style file paths?

Instead of converting the `compileStep.inputPath` can we convert the `indexFilePath` based on the detected platform? Or at least just convert the `compileStep.inputPath` just once.